tests/sample_projects/hidden_files/expected.toml

```diff
@@ -22,3 +22,57 @@ undeclared_deps = []
 
 # Declared dependencies which were never `import`ed from the code:
 unused_deps = []
+
+[experiments.ignore_nothing]
+description = "Run without ignores - everything is found."
+pyenvs = [""]  # Find all Python environments in project
```

Comment:
Why aren't the pyenvs under `.venvs/`detected if the dot ignore pattern is overridden?

Reply:
Oh, crap. This is poorly documented...

The `pyenvs = [""]` here is not the Python environments that we expect to find (i.e. the expected output), but rather the `--pyenv` values that we pass _into_ the analysis (that is, we're passing `""` which is appended onto the directory containing the sample project, meaning that we're searching through the entire sample project for Python envs). The Python environments that we end up finding are actually _not_ directly verified here at all...

Reply:
Oops